src/docfx/template/TemplateEngine.cs

```diff
@@ -36,15 +40,43 @@ private TemplateEngine(string templateDir, JsonSchema metadataSchema)
             var contentTemplateDir = Path.Combine(templateDir, "ContentTemplate");
             var schemaDir = Path.Combine(contentTemplateDir, "schemas");
 
+            Global = LoadGlobalTokens(contentTemplateDir);
             _templateDir = templateDir;
             _schemaDir = schemaDir;
             _liquid = new LiquidTemplate(templateDir);
             _js = new JavascriptEngine(contentTemplateDir);
-            Global = LoadGlobalTokens(contentTemplateDir);
             _schemas = Directory.Exists(schemaDir) ? Directory.EnumerateFiles(schemaDir, "*.schema.json", SearchOption.TopDirectoryOnly)
```

Comment:
consider extract a method here:

```suggestion
            _schemas = LoadSchemas(xxx)
```